validation-test/stdlib/Collection/LazyMapCollection.swift

```diff
@@ -0,0 +1,175 @@
+    LifetimeTracked(element.value, identity: element.identity)
+  },
+  extractValueFromEquatable: { (element: LifetimeTracked) in
+    MinimalEquatableValue(element.value, identity: element.identity)
+  }
+)
+
+CollectionTests.addRandomAccessCollectionTests (
+  makeCollection: { (elements: [LifetimeTracked]) in
+    MinimalCollection(elements: elements).lazy.map { $0 }
+  },
+  wrapValue: { (element: OpaqueValue<Int>) in
+    LifetimeTracked(element.value, identity: element.identity)
+  },
+  extractValue: { (element: LifetimeTracked) in
+    OpaqueValue(element.value, identity: element.identity)
+  },
+  makeCollectionOfEquatable: { (elements: [LifetimeTracked]) in
+    MinimalCollection(elements: elements).lazy.map { $0 }
+  },
+  wrapValueIntoEquatable: { (element: MinimalEquatableValue) in
+    LifetimeTracked(element.value, identity: element.identity)
+  },
+  extractValueFromEquatable: { (element: LifetimeTracked) in
+    MinimalEquatableValue(element.value, identity: element.identity)
+  }
+)
+
+CollectionTests.addRandomAccessCollectionTests (
+  makeCollection: { (elements: [LifetimeTracked]) in
+    MinimalBidirectionalCollection(elements: elements).lazy.map { $0 }
+  },
+  wrapValue: { (element: OpaqueValue<Int>) in
+    LifetimeTracked(element.value, identity: element.identity)
+  },
+  extractValue: { (element: LifetimeTracked) in
+    OpaqueValue(element.value, identity: element.identity)
+  },
+  makeCollectionOfEquatable: { (elements: [LifetimeTracked]) in
+    MinimalBidirectionalCollection(elements: elements).lazy.map { $0 }
+  },
+  wrapValueIntoEquatable: { (element: MinimalEquatableValue) in
+    LifetimeTracked(element.value, identity: element.identity)
+  },
+  extractValueFromEquatable: { (element: LifetimeTracked) in
+    MinimalEquatableValue(element.value, identity: element.identity)
+  }
+)
+
+CollectionTests.addRandomAccessCollectionTests (
+  makeCollection: { (elements: [LifetimeTracked]) in
+    MinimalRandomAccessCollection(elements: elements).lazy.map { $0 }
+  },
+  wrapValue: { (element: OpaqueValue<Int>) in
+    LifetimeTracked(element.value, identity: element.identity)
+  },
+  extractValue: { (element: LifetimeTracked) in
+    OpaqueValue(element.value, identity: element.identity)
+  },
+  makeCollectionOfEquatable: { (elements: [LifetimeTracked]) in
+    MinimalRandomAccessCollection(elements: elements).lazy.map { $0 }
+  },
+  wrapValueIntoEquatable: { (element: MinimalEquatableValue) in
+    LifetimeTracked(element.value, identity: element.identity)
+  },
+  extractValueFromEquatable: { (element: LifetimeTracked) in
+    MinimalEquatableValue(element.value, identity: element.identity)
+  }
+)
+
+// Test sequence instances and iterators.
+CollectionTests.test("LazyMapCollection instances") {
+  do {
+    let expected = ["convent", "conform", "constrict", "condone"]
+    let base = ["vent", "form", "strict", "done"]
+    checkSequence(expected, base.lazy.map { "con" + $0 })
```

Comment:
It would be best if they were, yes!  Then you'd only need one test, because you would not care if the elements are strings or numbers.